gradio/components/base.py

```diff
@@ -240,7 +240,7 @@ def __str__(self):
         return self.__repr__()
 
     def __repr__(self):
-        return f"{self.get_block_name()}"
+        return f"<{self.__class__.__name__} object> with config: {self.get_config()}"
```

Comment:
Made this change due in part to this internal discussion: https://huggingface.slack.com/archives/C02QZLG8GMN/p1708388337912479?thread_ts=1708385877.550569&cid=C02QZLG8GMN

Reply:
Do we need to implement custom `__str__` and `__repr__`? Dumping the whole config to the terminal can be unwieldy and the built-in methods make it clear it's an object and not just a type

<img width="564" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/d31fe6f3-8c9a-40de-8bd8-357ce45600af">

Reply:
Ok I don't feel strongly at all, will just eliminate them